src/Xablu.WebApiClient/CrossRestApiClient.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using Xablu.WebApiClient.Exceptions;
+
+namespace Xablu.WebApiClient
+{
+    public class CrossRestApiClient
+    {
+        private static Action<RestApiClientOptions> _configureRestApiClient;
+        private static Lazy<IRestApiClient> _restApiClientImplementation = new Lazy<IRestApiClient>(() => CreateRestApiClient(), System.Threading.LazyThreadSafetyMode.PublicationOnly);
+
+        public static bool IsSupported => _restApiClientImplementation.Value == null ? false : true;
```

Comment:
Whats the use case for this bool?

Reply:
This is an bool from the old bait & switch method and should be removed here. 

In the old setup, one would always get at least the PCL library installed (even if your Platform was not supported). You could then use this boolean to make sure your platform was really supported. 

Now with Multitargeting you get an error while trying to install the package through nuget, so the use of this boolean became obsolete

Reply:
There are also two sample solutions why is that?

Reply:
Because I forgot to delete the old one ;)